debugAdapterProtocol.json

```diff
@@ -14,7 +14,7 @@
 			"properties": {
 				"seq": {
 					"type": "integer",
-					"description": "Sequence number (also known as message ID). For protocol messages of type 'request' this ID can be used to cancel the request."
+					"description": "Sequence number of the message (also known as message ID). It is used to order requests, responses, and events, and to associate requests with their corresponding responses. For protocol messages of type 'request' the sequence number can be used to cancel the request."
```

Comment:
I don't think the `seq` is used for ordering--is it? Would a debugger that selects a random int each time for its `seq` behavior differently than one that uses an incremental seq?

Reply:
The `seq` property, its name and semantics were inherited from the old V8 debugger protocol.
I'm pretty sure `seq` is not used for ordering in VS code, but another client or DA might rely on the "sequence" semantics implied by the name. So we can't just change the spec to "although called `sequence`, a client can use unique numbers in random order".

Reply:
Perhaps we should add something like

> The `seq` for the first message sent by a client or debug adapter is 1, and the `seq` of each subsequent message is 1 greater than the previous message.